docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -151,21 +153,27 @@ To demonstrate this flexibility, consider this nonsensical quantile on `span.htt
 { name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
 ```
 
-### The `compare` function
 
-This adds a new metrics function `compare` which is used to split the stream of spans into two groups: a selection and a baseline.
+
+## The `compare` function
+
+The `compare` function is used to split the stream of spans into two groups: a selection and a baseline.
 It returns time-series for all attributes found on the spans to highlight the differences between the two groups.
-This is a powerful function that is best understood by looking at example outputs below:
+This is a powerful function that is best understood by looking at example outputs below.
```

Comment:
You could also say this function powers the "Compare" tab in Explore Traces.

Reply:
Honestly I think that's a better way to understand what it does. =)

Reply:
```suggestion
This is a powerful function that is best understood by using the [**Compare** tab in Explore Traces](https://grafana.com/docs/grafana/<GRAFNA_VERSION>/explore/simplified-exploration/traces/investigate/#comparison). 
You can also under this function by looking at example outputs below.
```

Reply:
I've added a link to the Comparison tab in Explore Traces doc.

Reply:
Done.